lib/client/rpc/market-clients/ammMarkets.ts

```diff
@@ -78,82 +74,106 @@ export class FutarchyAmmMarketsRPCClient implements FutarchyAmmMarketsClient {
     }
   }
 
+  validateAddLiquidity(
+    ammMarket: AmmMarket,
+    quoteAmount: number,
+    maxBasAmount: number
```

Comment:
`maxBaseAmount` is likely what you wanted to pass as the name.

Reply:
Whoopsie fixed this